apps/prairielearn/src/pages/authLogout/authLogout.ts

```diff
@@ -22,9 +22,6 @@ router.get(
       setCookie(res, ['pl_disable_auto_authn', 'pl2_disable_auto_authn'], '1');
     }
 
-    // Hold-over from the old express-session implementation
-    clearCookie(res, 'connect.sid');
-
```

Comment:
I just made this change. Any stored cookies would have expired long ago, and we're certainly not reading this cookie anymore.